src/bot/commands/delete.ts

```diff
@@ -0,0 +1,147 @@
+import Command from '../lib/command';
+import {
+  APIEmbed,
+  ApplicationCommandOptionType,
+  ApplicationCommandType,
+  bold,
+  ButtonStyle,
+  ChatInputCommandInteraction,
+  codeBlock,
+  ComponentType,
+  Message,
+  roleMention,
+  TextChannel
+} from 'discord.js';
+import {
+  buildJokeDisplay,
+  interactionInfo,
+  interactionProblem,
+  interactionValidate,
+  isEmbedable,
+  waitForInteraction
+} from '../utils';
+import { CategoriesRefs } from '../../typings';
+import { jokeById, jokeByQuestion } from '../../controllers';
+import { Colors, logsChannelId } from '../constants';
+import Jokes from '../../jokes';
+import prisma from '../../prisma';
+import { stripIndents } from 'common-tags';
+
+export default class DeleteCommand extends Command {
+  constructor() {
+    super({
+      name: 'delete',
+      description: 'Supprimer une blague',
+      type: ApplicationCommandType.ChatInput,
+      options: [
+        {
+          type: ApplicationCommandOptionType.String,
+          name: 'recherche',
+          description: 'ID ou question de la blague',
+          required: true
+        }
+      ]
+    });
+  }
+
+  async run(interaction: ChatInputCommandInteraction<'cached'>) {
+    if (interaction.user.id !== process.env.BOT_USER_OWNER) {
```

Comment:
```suggestion
    if (interaction.user.id !== process.env.BOT_OWNER_ID) {
```
Je pense que ce serait plus clair de rénover la variable d'environnement BOT_OWNER_ID par ce que comme ça on peut facilement voir que c'est l'id de l'owner du bot... par ce que avant, c'était lisible, mais pas aussi clair je trouve

Reply:
Je suis d'accord 👍

Reply:
Pas faux...